.travis.yml

```diff
@@ -58,36 +62,50 @@ jobs:
         - export HW=x86
         - sed -i 's/java_arch=32-bit$/java_arch=64-bit/' build-image/openhabian.conf
         - docker build -t openhabian/${HW}-openhabian -f Dockerfile.openhabian-x86 .
+        - echo -e "\n\e[36mopenHABian test installation 64bit Azul Enterprise Java on virtual $HW ...\n"
         - docker run --name "openhabian-${HW}" -d openhabian/${HW}-openhabian;
           docker exec -i "openhabian-${HW}" bash -c \
             'VIRT=$(sudo virt-what); echo -e "\n\e[37mTesting openhabian install on ${VIRT} ...\n"; ./build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local'
     - env: arm32hf
+      arch: arm64
       script:
-        - docker run --rm --privileged multiarch/qemu-user-static:register --reset
-        - export HW=rpi2
+        - export HW=rpi01
+#        - docker run --rm --privileged multiarch/qemu-user-static:register --reset
```

Comment:
pls explain intention of commenting out or remove

Reply:
ok